### PR TITLE
Morpheus "Apollo" gentx preparation

### LINKED ISSUE #5: Reduce unbonding period in the next testnet
Currently the unbonding period is set to the default 21-day. Undelegation and redelegation have to be locked in 21 days. We should reduce this to enable more rapid testing.

### LINKED ISSUE #43: Fix supply inside genesis transaction
Currently the `udaric` supply inside the genesis is not correct. We should update it with the proper value

### LINKED ISSUE #201: Add genesis accounts
Currently, genesis transactions are all added without the associated genesis accounts. We should give each delegator a default amount of `10000000udaric` (`10 Daric`).

### LINKED ISSUE #274: Add faucet account to Apollo Genesis
Remember to add faucet account into the Apollo genesis.

### PR DESCRIPTION
**Pull request by rmontagnin**
This PR prepares the final `genesis.json` file for the upcoming `morpheus-apollo-1` testnet. The following changes have been applied: 

1. Updated the `x/auth` accounts list, to include all the genesis validators' delegators. 
2. Updated the `x/bank` balances list, to give each of the genesis validators' delegator `10000000udaric` (10 Daric). 
3. Run the `desmos collect-gentxs` command to collect all the valid genesis transactions. 
4. Increased `max_validators` to `150`
5. Added vesting account

Closes #5, closes #43, closes #201, closes #274

## Note on the genesis transactions
There have been a total of 193 submitted genesis transactions. Of all of these, I deleted

- all the invalid ones (empty body, or created with an old `desmos` version)
- all the duplicate ones (either same validator, same delegator or both)
- all the ones that came from people that did not submit the validators' survey in time
- all the ones that came from people that submit wrong values inside the validators' survey (eg. wrong operator address)

### CODE REVIEW

File: morpheus-apollo-1/genesis.json [hunk 1]
```diff
@@ -30,51 +30,11142 @@
...
+          "sequence": "0"
+        },
+        {
+          "@type": "/cosmos.auth.v1beta1.BaseAccount",
+          "address": "desmos1p35njvw060u2tjw73ncvtgapmlw7ry9fqw2ja2",
+          "pub_key": null,
+          "account_number": "0",
+          "sequence": "0"
+        },
+        {
```
Comment (leobragaz) [inline morpheus-apollo-1/genesis.json:475]: remove from here to 481

File: morpheus-apollo-1/genesis.json [hunk 2]
```diff
@@ -30,51 +30,11142 @@
...
+                "commission": {
+                  "max_change_rate": "0.010000000000000000",
+                  "max_rate": "1.000000000000000000",
+                  "rate": "0.100000000000000000"
+                },
+                "delegator_address": "desmos1nj3anmeemlgt5ym92f283wzlq3zfr3327x2pfe",
+                "description": {
+                  "details": "test-drill",
+                  "identity": "",
+                  "moniker": "leobragaz",
```
Comment (leobragaz) [inline morpheus-apollo-1/genesis.json]: remove this validator

**Review by leobragaz — CHANGES_REQUESTED** [review]

**Review by kwunyeung — CHANGES_REQUESTED** [review]
I can successfully start a node if I move the genesis time to past time. The current 144 accounts and genesis txs are correct.

Let's update the `max_validators` to `150`.

File: morpheus-apollo-1/genesis.json [hunk 3]
```diff
@@ -30,51 +30,11142 @@
...
+        },
+        {
+          "address": "desmos1ct2uu96e80xq3d6x4ual4d4sxna678p9qxaryt",
+          "coins": [
+            {
+              "denom": "udaric",
+              "amount": "10000000"
+            }
+          ]
+        }
```
Comment (kwunyeung) [inline morpheus-apollo-1/genesis.json:2554]: Add bank balances for the vesting account
```suggestion
        },
        {
          "address": "desmos1snmetx9sfrmznr4z6d3lnwlyvwu36sgcpzzhlr",
          "coins": [
            {
              "denom": "udaric",
              "amount": "300000000000"
            }
          ]
        }
```

File: morpheus-apollo-1/genesis.json [hunk 4]
```diff
@@ -30,51 +30,11142 @@
...
+          "account_number": "0",
+          "sequence": "0"
+        },
+        {
+          "@type": "/cosmos.auth.v1beta1.BaseAccount",
+          "address": "desmos1ct2uu96e80xq3d6x4ual4d4sxna678p9qxaryt",
+          "pub_key": null,
+          "account_number": "0",
+          "sequence": "0"
+        }
```
Comment (kwunyeung) [inline morpheus-apollo-1/genesis.json:1154]: Add a new vesting account with vesting period
```suggestion
        },
        {
          "@type": "/cosmos.vesting.v1beta1.PeriodicVestingAccount",
          "base_vesting_account": {
            "base_account": {
              "address": "desmos1snmetx9sfrmznr4z6d3lnwlyvwu36sgcpzzhlr",
              "pub_key": null,
              "account_number": "0",
              "sequence": "0"
            },
            "original_vesting": [
              {
                "denom": "udaric",
                "amount": "30000000000"
              }
            ],
            "delegated_free": [],
            "delegated_vesting": [],
            "end_time": "1619766000"
          },
          "start_time": "1619506800",
          "vesting_periods": [
            {
              "length": "86400",
              "amount": [
                {
                  "denom": "udaric",
                  "amount": "10000000000"
                }
              ]
            },
            {
              "length": "86400",
              "amount": [
                {
                  "denom": "udaric",
                  "amount": "10000000000"
                }
              ]
            },
            {
              "length": "86400",
              "amount": [
                {
                  "denom": "udaric",
                  "amount": "10000000000"
                }
              ]
            }
          ]
        }
```

**Review by kwunyeung — CHANGES_REQUESTED** [review]

File: morpheus-apollo-1/genesis.json (hunk 1 above)
Comment (rmontagnin) [reply morpheus-apollo-1/genesis.json:475]: Done

File: morpheus-apollo-1/genesis.json (hunk 2 above)
Comment (rmontagnin) [reply morpheus-apollo-1/genesis.json]: Done

**Review by leobragaz — APPROVED** [review]
🚀

**Pull request merged**